bitnami/minio/values.yaml

```diff
@@ -572,6 +578,12 @@ volumePermissions:
     ##
     limits: {}
     requests: {}
+  ## Init container' Security Context
+  ## Note: the chown of the data folder is done to containerSecurityContext.runAsUser
+  ## and not the below volumePermissions.securityContext.runAsUser
+  ##
+  securityContext:
```

Comment:
I would change it to containerSecurityContext for consistency

Reply:
Thank you for your feedback, I changed it to containerSecurityContext